src/interactive-window/interactiveWindow.ts

```diff
@@ -197,14 +198,12 @@ export class InteractiveWindow implements IInteractiveWindowLoadable {
             };
             // When connecting, we need to update the sys info message
             this.updateSysInfoMessage(this.getSysInfoMessage(metadata, SysInfoReason.Start), false, sysInfoCell);
-            const kernel = await KernelConnector.connectToKernel(
-                controller,
-                metadata,
-                this.serviceContainer,
-                { resource: this.owner, notebook: this.notebookDocument },
+            const vscController = Array.from(this.controllerRegistrations.values).find(
```

Comment:
isn't `this.controllerRegistrations.values` already an array? (making `Array.from` unnecessary)

Reply:
Oh yes, at one point this was pointing to some dictionary, i think i forgot this changed.